### PR TITLE
Sprint 7 unit tests

### PR DESCRIPTION
**Pull request by AbdulSir**
### We have reached a test coverage of 83.47% statement, and 73.29% branch

### CODE REVIEW

**Comment by AbdulSir** [comment]
image
image

**Review by purplesmurf1998 — APPROVED** [review]
Tests look good and we have a good coverage.

File: backend/controllers/commentsController.js [hunk 1]
```diff
@@ -23,12 +23,12 @@ exports.getComments = asyncHandler(async (req, res, next) => {
 // @route       POST /api/v1/comments/vehicle/:vehicleId
 // @access      Private
 exports.createComment = asyncHandler(async (req, res, next) => {
-  
+  console.log("TESTTEST");
```
Comment (TZnoj) [inline backend/controllers/commentsController.js:26]: This console log can be removed

Comment (AbdulSir) [reply backend/controllers/commentsController.js]: Thank you for flagging this. It's been removed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/91/commits/3d6230aa52a906de840c32c937d900b2d8777f78)

**Review by TZnoj — APPROVED** [review]
Great work on the test coverage, i see that a sizable amount of tests were added

**Pull request merged**